src/ng/compile.js

```diff
@@ -2455,6 +2469,24 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
+    function addEventBindingDirective(directives, eventName, isEventDirective) {
+      if (!isEventDirective) return;
+
+      directives.push({
+        compile: function(tElement, tAttr) {
```

Comment:
doing this in compile seems dangerous for directives which use `replace`? Or element transclusion?